options/locale/locale_en-US.ini

```
@@ -886,6 +886,7 @@ migrate_items_releases = Releases
 migrate_repo = Migrate Repository
 migrate.clone_address = Migrate / Clone From URL
 migrate.clone_address_desc = The HTTP(S) or Git 'clone' URL of an existing repository
+migrate.github_token_desc = You can add a comma-separated list of tokens here to speed up the migration. We use the GitHub API to migrate, and hence its rate limits apply per token. Whitespace around a token will be ignored. Please be aware that adding too many tokens (abusing this feature) might get your account banned. According to our tests, <x> tokens should not present a problem.
```

I would just not mention a number ... is it 1,2 or 100 ...
```suggestion
migrate.github_token_desc = You can add a comma-separated list of tokens here to speed up the migration. We use the GitHub API to migrate, and hence its rate limits apply per token. Whitespace around a token will be ignored. Please be aware that adding too many tokens (abusing this feature) might get your account banned.
```